crates/bevy_wgpu/src/wgpu_renderer.rs

```diff
@@ -19,13 +19,15 @@ pub struct WgpuRenderer {
 impl WgpuRenderer {
     pub async fn new() -> Self {
         let instance = wgpu::Instance::new(wgpu::BackendBit::PRIMARY);
-        let adapter = instance
+        let adapter = if let Some(a) = instance
```

Comment:
I think we can make this a little more terse like so:
```rust
instance
            .request_adapter(&wgpu::RequestAdapterOptions {
            .request_adapter(&wgpu::RequestAdapterOptions {
                power_preference: wgpu::PowerPreference::Default,
                power_preference: wgpu::PowerPreference::Default,
                compatible_surface: None,
                compatible_surface: None,
            })
            }).await.expect("Unable to find a GPU! Make sure you have installed required drivers!")
```

Reply:
I'll take a look at it tomorrow/later today, as it's 0:24 and I've got work in about 5 hours 😅
Thanks!

Reply:
Thank you :)